evalml/utils/gen_utils.py

```diff
@@ -272,6 +273,25 @@ def is_all_numeric(df):
     return True
 
 
+def _convert_to_woodwork_structure(data):
+    """
+    Takes input data structure, and if it is not a Woodwork data structure already, will convert it to a Woodwork DataTable or DataColumn structure.
+    """
+    if isinstance(data, ww.DataTable) or isinstance(data, ww.DataColumn):
+        return data
+    # Convert numpy data structures to pandas data structures
+    if isinstance(data, list):
+        data = np.array(data)
+    if isinstance(data, pd.api.extensions.ExtensionArray) or (isinstance(data, np.ndarray) and len(data.shape) == 1):
```

Comment:
@angela97lin cool I didn't realize `ExtensionArray` was specifically 1d, but I see [the pandas docs mention it](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.api.extensions.ExtensionArray.html). Looks good